sale_mrp_link/README.rst

```diff
@@ -14,13 +14,13 @@ Sale MRP Link
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
-    :target: https://github.com/OCA/sale-workflow/tree/10.0/sale_mrp_link
+    :target: https://github.com/OCA/sale-workflow/tree/12.0/sale_mrp_link
```

Comment:
@NachoAlesLopez 11.0, not 12.0
Everywhere

Reply:
Done!